Dockerfile

```diff
@@ -1,31 +1,32 @@
-# --------------------------------------------------------------------------
-# This is a Dockerfile to build an Ubuntu 14.04 Docker image with
-# pymssql and FreeTDS
-#
-# Use a command like:
-#     docker build -t pymssql/pymssql .
-# --------------------------------------------------------------------------
+FROM  ubuntu:18.04
+MAINTAINER  Tyler French tylerfrench2@gmail.com (@frenchtoasters)
 
-FROM  orchardup/python:2.7
-MAINTAINER  Marc Abramowitz <marc@marc-abramowitz.com> (@msabramo)
+# Install dependices
+RUN apt-get update && apt-get install -y \
+    unixodbc \
+    unixodbc-dev \
+    unixodbc-bin \
+    libodbc1 \
+    odbcinst1debian2 \
+    tdsodbc \
+    php-odbc \
+    python3 \
+    python3-pip
 
-# Install apt packages
+# Install FreeTDS
 RUN apt-get update && apt-get install -y \
     freetds-bin \
     freetds-common \
-    freetds-dev
+    freetds-dev \
+    libct4 \
+    libsybdb5
 
-RUN pip install Cython
-RUN pip install ipython
-RUN pip install SQLAlchemy
-RUN pip install pandas
-RUN pip install Alembic
+# Install pip dependencies
+RUN pip3 install Cython
 
 # Add source directory to Docker image
-# Note that it's beneficial to put this as far down in the Dockerfile as
-# possible to maximize the chances of being able to use image caching
 ADD . /opt/src/pymssql/
```

Comment:
@frenchtoasters What's the use case for this Docker file?  Based on the answer to that question, I wonder if:

1. This should be a volume mapping instead so the image doesn't need to be rebuilt every time you change a pymssql file.  In that case, we should also install with `pip install -e`.
2. We should install from the latest released version on pypi

Reply:
The use case for me on this Dockerfile was to use it as a base for a project that used pymssql, while also allowing for changes to source code if needed prior to build. 

1. You are correct that the volume mapping would save build time, I chose to copy the files into the build for simplicity of the build command. Installing with `pip install -e` might be well suited for this while developing but I was just using this as a base to build a child container from. However im not that well versed on the use cases for `pip install -e`, what about it makes you think it should be installed that way?

2. I dont think it should be installed via pypi in this image because that would not allow for modifications of the source and testing changes.